storage.rst

```diff
@@ -4,36 +4,64 @@
 Storage
 =======
 
-As default, logs are written in a volatile in-memory directory to prevent errors on the root file system in case of failure.
-The memory storage capacity is limited and the oldest logs are automatically removed as time passes to limit memory occupancy. However, if you reboot the system or your system fails, you may lose all of your logs.
+Starting from version 8.6, NethSecurity automatically ensures that system logs are saved to a persistent storage partition.  
```

Comment:
Logs are not persistent across reboots on installations on virtual machines without additional storage.
We might say "tries to ensure...if storage is available".